// const errorhandler = (fun) => {
//   return (call, callback) => {
//     try {
//       fun(call, callback);
//     } catch (err) {
//       return err;
//     }
//   };
// };

// export default errorhandler;
